angrmanagement/ui/menus/disasm_insn_context_menu.py

```diff
@@ -1,13 +1,19 @@
 from functools import partial
 from typing import Callable
+from typing import TYPE_CHECKING
 
 from ...config import Conf
 from .menu import Menu, MenuEntry, MenuSeparator
 
+if TYPE_CHECKING:
+    from ...ui.views.disassembly_view import DisassemblyView
 
 class DisasmInsnContextMenu(Menu):
-    def __init__(self, disasm_view):
-        super(DisasmInsnContextMenu, self).__init__("", parent=disasm_view)
+    """
+    Dissembly Instruction's Context Menu
+    """
```

Comment:
You don't seem to have taken my advice on how to add useful class docstrings. If you don't have anything good to add to the documentation, just leave it blank and we'll merge it with failing lint.

Reply:
I just forget this one.